magento/magento2
# [New Relic] Making system configs dependent by Enabled field

### Description (*)
This PR makes all the New Relic system configs dependent by `Enable New Relic Integration` field.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
1. Navigate to `Store / Configuration / New Relic Reporting [General]`
2. All the configuration fields should be hidden if `Enable New Relic Integration` -> `No`

### Questions or comments


### Contribution checklist (*)
